conf/modules.config

```diff
@@ -620,6 +639,9 @@ process {
         ]
     }
 
+    withName: 'CONCOCT_CUTUPFASTA' {
+        ext.args = '-c 10000 -o 0 --merge_last'
```

Comment:
I vote following @GallVp 's suggestion - they should be customisable parameters with these two values as the defaults